scripts/cluster-install/openstack_connection.py

```diff
@@ -0,0 +1,127 @@
+import time
+import traceback
+import openstack
+
+
+def _get_client_config(config):
+    client_config = {
+        'auth_url': config['auth_url'],
+        'username': config['username'],
+        'password': config['password'],
+        'region_name': config['region_name'],
+        'project_name': config['tenant_name'],
+        'user_domain_name': 'Default',
+        'project_domain_id': 'Default'
+    }
+    return client_config
+
+
+def _get_resource_config(config, name):
+    resource_config = {
+        'name': name,
+        'image_id': '70de1e0f-2951-4eae-9a8f-05afd97cd036',  # centos image_id
+        'flavor_id': '3',  # m1.medium id
```

Comment:
This can actually all be done on small VMs, which might be nice to do so that we use less resources by default.
This could also be configurable and just default to the small VM (ID is 2 I think?).
To make it work you also need to add the following lines to all the configs:
validations:
  minimum_required_total_physical_memory_in_mb: 1024

Reply:
Done.